client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -90,6 +92,25 @@ const UserPage = createClass({
 		  </td>;
 	},
 
+	handleFilterTextChange : function(e){
+		this.setState({
+			filterString : e.target.value
+		});
+		return;
+	},
+
+	renderFilterOption : function(){
+		return <td>
+			<label>
+				Filter Title:
+				<input
+					type='text'
+					onChange={this.handleFilterTextChange}
```

Comment:
`onChange` only fires after the input field loses focus — it would be better to have `onInput' here for immediate responses as they type (otherwise they might go looking for a "do search" button, or think they need to press <kbd>ENTER</kbd> to submit etc.

Reply:
This works differently in React - `onChange` does not require loss of focus to fire, making it functionally identical to `onInput`.